subsys/nrf_security/configs/nrf-config.h.template

```diff
@@ -10,6 +10,10 @@
  */
 #ifndef MBEDTLS_CONFIG_FILE_H
 #define MBEDTLS_CONFIG_FILE_H
+/****************************************************************/
```

Comment:
Why this line? Plus there's a typo in this commit's message.

Reply:
This commit will now get removed since with the updates in nrf_security it is not needed.